Client/Frontend/Browser/URLBarView.swift

```diff
@@ -299,25 +300,25 @@ class URLBarView: UIView {
     }
 
     func updateTabCount(count: Int) {
-
         // make a 'clone' of the tabs button
         let newTabsButton = InsetButton()
+        newTabsButton.setTranslatesAutoresizingMaskIntoConstraints(false)
```

Comment:
Yup looks like SnapKit turns this off when you setup constraints using snp_makeConstraints. I'll remove this.